arbitrary-users-patch/check_sidecar_image_digests.sh

```diff
@@ -14,19 +14,27 @@
 
 set -e
 
+
```

Comment:
`NO_OP` variable not defined as "false". Not needed, but also bad form to implicitly create a variable and then check its contents later.